dll/win32/setupapi/install.c

```diff
@@ -693,7 +693,12 @@ static BOOL register_dlls_callback( HINF hinf, PCWSTR field, void *arg )
         if (!SetupGetIntField( &context, 4, &flags )) flags = 0;
 
         /* get timeout */
+#ifdef __REACTOS__
+        /* "11,,cmd.exe,,,/K dir" means default timeout, not a timeout of zero */
```

Comment:
:+1:

Reply:
Excellent. Can this be proposed to Wine upstream as well?